Package.swift

```diff
@@ -13,41 +13,32 @@ let package = Package(
     ],
     products: [
         // Products define the executables and libraries a package produces, and make them visible to other packages.
-        .library(
```

Comment:
I think we should revert this part.
I just tested with the latest version of multipart-kit - the serialisation is not correct for our purposes out of the box.
If MultipartKit requires concurrency-related modifications, we can pull their upstream and re-apply our tweaks (or re-visit doing this manually). Anyway, we can do this once the rest of TootSDK is cleared for concurrency.